Cargo.toml

```diff
@@ -67,7 +68,7 @@ criterion = "0.5.0"
 default = ["rayon", "default-formats"]
 
 # Format features
-default-formats = ["avif", "bmp", "dds", "exr", "ff", "gif", "hdr", "ico", "jpeg", "png", "pnm", "qoi", "tga", "tiff", "webp"]
+default-formats = ["avif", "bmp", "dds", "exr", "ff", "gif", "hdr", "ico", "jpeg", "pcx", "png", "pnm", "qoi", "tga", "tiff", "webp"]
```

Comment:
I would definitely prefer this to be off by default. Especially that Cargo's feature merging means it's sometimes pretty difficult to turn off the default features.

This is a truly obsolete format, so for nearly every user of this library it will be nothing but overhead that adds to the compilation time and bloats executables.

Reply:
:thumbsup: - https://github.com/image-rs/image/pull/2364/commits/84cd6041098cf74306178f09b481469aea9793e7 disables it by default.